src/data/full-quiz.js

```diff
@@ -2,38 +2,36 @@ const fullQuiz = [
   
    {
    Question:
-     "Which operator is used to find remainder in Python?",
-    Answer: "%",
-    Distractor1: "*",
-    Distractor2: "X",
-    Distractor3: "/",
-    Distractor4: "#",
+     "Which method can be used to replace parts of a string?",
+    Answer: "replace()",
+    Distractor1: "switch()",
+    Distractor2: "repl()",
+    Distractor3: "find()",
+    Distractor4: "replacString()",

```

Comment:
Please delete distractor4

Reply:
I have done the
Thank you so much for your guidance